docs/helpers/testing.md

```diff
@@ -44,7 +44,7 @@ describe('Search Endpoint', () => {
   // Create the test client from the app instance
   const client = testClient(app)
 
-  it('should return search results', async () => {
+  test('should return search results', async () => {
```

Comment:
I think it's better to use `it` instead of `test` since the message starts with `should`. So please replace the import statement:

https://github.com/honojs/website/blob/a047eeeb2c6e0cad482e8b46a5f2b266368d836c/docs/helpers/testing.md?plain=1#L40